README.md

```diff
@@ -22,3 +22,7 @@ Scratchy, application de tchat projet LPH.
   - [📸 Frontend diagram](doc/front-uml-class-diagram.png)
 
 
+## Tools
+
+ - :wrench: tools documentation directory is [here](doc/tools)
+ - python linter: [flake8](doc/tools/flake8.md)
```

Comment:
a 404 error with links. links must be a relative path

Reply:
??? the path is relative and it work perfectly in the browser what are you talking about ?